cg/cli/demultiplex/sample_sheet.py

```diff
@@ -53,9 +53,15 @@ def validate_sample_sheet(
     LOG.info(
         f"Validating {sheet} as a {flow_cell.sequencer_type} {bcl_converter} sample sheet",
     )
-    sheet: Path = Path(str(sheet))
+    sample_sheet_path = Path(str(sheet))
+    sample_sheet_content: List[List[str]] = ReadFile.get_content_from_file(
+        file_format=FileFormat.CSV, file_path=sample_sheet_path
+    )
     try:
-        get_sample_sheet_from_file(infile=sheet, flow_cell_sample_type=flow_cell.sample_type)
+        get_validated_sample_sheet(
+            sample_sheet_content=sample_sheet_content,
+            sample_type=flow_cell.sample_type
+        )
```

Comment:
This cli command could also just be:
```python
@sample_sheet_commands.command(name="validate")
@click.argument("sheet", type=click.Path(exists=True, dir_okay=False))
@click.pass_obj
def validate_sample_sheet(sheet: click.Path):
    """Validate that the sample sheet can be parsed."""
    sample_sheet_path = Path(str(sheet))
    try:
        get_sample_sheet_from_file(sample_sheet_path)
    except ValidationError as error:
        LOG.warning(error)
        raise click.Abort from error
    LOG.info("Sample sheet passed validation")
```
What do you think? I kept the old behaviour since this PR is only intended to refactor logic - not change the behaviour.

Reply:
I like short and clear CLI commands like this one. I vote for the alternate option